### DISCUSSION TITLE
Missing Database Column named new_geom in the core_delta table.

### DISCUSSION THREAD

**Discussion by mahmoodi-source** [opengisch/QField#4985] [Bugs]
We are facing this problem with our self-hosted QfieldCloud server (V0.23.2). From one side (QGIS), we can upload the project to the server. But from Qfield, the new changes cannot be uploaded back to the server, and it gives an error. 
So, we have manually added a new column named new_geom to the table core_delta, and it start working perfectly. Does anybody know why this column was missing?

**Comment by suricactus** [comment]
Seems you had a problem with the migrations. This column is defined in https://github.com/opengisch/qfieldcloud/blob/master/docker-app/qfieldcloud/core/migrations/0051_auto_20211125_0444.py#L20-L33 .

**Reply by mahmoodi-source** [reply]
Thanks for your good inofrmation.
